APIManager.js

```diff
@@ -0,0 +1,165 @@
+//This is the class that will manage all your APIs
+
+class APIManager
+{
+    constructor()
+    {
+        this.data = {};
+    }
+
+    loadData()
+    {
+        this.data.users = new UsersAPI().loadData();
+        this.data.quotes = new QuotesAPI().loadData();
+        this.data.bacons = new BaconsAPI().loadData();
+        this.data.pokemons = new PokemonsAPI().loadData();
```

Comment:
there's no need to create a different class for each external API, especially if there is only one method in each of those classes.
If there was more functionality for each class, then maybe